osier/models/dispatch.py

```diff
@@ -78,30 +107,144 @@ class DispatchModel():
     model : :class:`pyomo.environ.ConcreteModel`
         The :mod:`pyomo` model class that converts python code into a
         set of linear equations.
-    upperbound : float
+    objective : float
+        The result of the model's objective function. Only instantiated
+        after :meth:`DispatchModel.solve()` is called.
+    n_timesteps : int
+        The number of timesteps in the model.
+    upper_bound : float
         The upper bound for all decision variables. Chosen to be equal
         to the maximum capacity of all technologies in :attr:`tech_set`.
+    storage_upper_bound : float
+        The upper bound for storage decision variables.
+    penalty : float
+        The penalty applied to the objective function to eliminate
+        simultaneous charging and discharging. Users may need to tune
+        this parameter. Default is 1.0.
+
```

Comment:
Ideally, it wouldn't need to be tuned. It's available *just in case* it does.

Reply:
I think that needs to be included in the docs somewhere.

Reply:
What would you suggest beyond "may need to be tuned?"

Reply:
Increasing it would increase the objetive fxn, discouraging storage, and decreasing it might produce simultaneous charging and discharging. Im hedging because I havent't tried it on an actual problem so I can't say when it would need to be tuned. You can create an issue about this if you think it needs to be addressed in the future! Perhaps with a theory page.

Reply:
I think a theory page would be good place for this. I'll create a new issue about it and won't let this stop this PR from getting approved and merged.